scout/parse/variant/clnsig.py

```diff
@@ -5,65 +5,71 @@
 
 def parse_clnsig(variant, transcripts=None):
     """Get the clnsig information
-    
+
     The clinvar format has changed several times and this function will try to parse all of them.
-    The first format represented the clinical significance terms with numbers. This was then 
-    replaced by strings and the separator changed. At this stage the possibility to connect review 
-    stats to a certain significance term was taken away. So now we can only annotate each 
+    The first format represented the clinical significance terms with numbers. This was then
+    replaced by strings and the separator changed. At this stage the possibility to connect review
+    stats to a certain significance term was taken away. So now we can only annotate each
     significance term with all review stats.
-    Also the clinvar accession number are is some cases annotated with the info key CLNACC and 
+    Also the clinvar accession numbers are in some cases annotated with the info key CLNACC and
     sometimes with CLNVID.
+    This function parses also Clinvar annotations performed by VEP (CSQ field, parsed transcripts required)
 
     Args:
         variant(cyvcf2.Variant)
-        acc(str): The clnsig accession number, raw from vcf
-        sig(str): The clnsig significance term, raw from vcf
-        revstat(str): The clnsig revstat, raw from vcf
         transcripts(iterable(dict))
 
     Returns:
         clnsig_accsessions(list(dict)): A list with clnsig accessions
     """
+
     transcripts = transcripts or []
     acc = variant.INFO.get("CLNACC", variant.INFO.get("CLNVID", ""))
     sig = variant.INFO.get("CLNSIG", "").lower()
     revstat = variant.INFO.get("CLNREVSTAT", "").lower()
 
-    clnsig_accsessions = []
+    clnsig_accessions = []
 
-    if not acc:
-        if transcripts:
+    if not acc and transcripts:
```

Comment:
I trust you, but that line is beyond my knowledge of python operator precedence. I would have had to throw in a parenthesis - or google it. I guess `not` binds harder / has higher precedence?

Reply:
I'll add the parentheses! :)